try:
    print("This is a try statement.") # Note the error. This will not execute at runtime.
except:
    print("This is an exception.") # This will execute at runtime.
else:
    print(2+2)